R/fin_year.R

```diff
@@ -5,9 +5,9 @@
 #'
 #' @details The PHS accepted format for financial year is yyyy/yy e.g. 2017/18.
 #'
-#' @param date A date which must be supplied with \code{Date} class. The
-#' functions as.Date() or lubridate::dmy() are examples of functions that can
-#' be used to change a variable to date class.
+#' @param date A date which must be supplied with \code{Date} or \code{POSIXct}
+#' class. The functions as.Date() or lubridate::dmy() as.POSIXct() are examples
```

Comment:
Should this be "...as.date(), lubridate::dmy() or as.POSIXct()..."?

Reply:
Yes, good catch! Will update and push.